virttest/qemu_devices/qcontainer.py

```diff
@@ -144,8 +144,9 @@ def get_qmp_cmds(qemu_binary, workaround_qemu_qmp_crash=False):
                              allow_output_check="combined",
                              shell=True,
                              verbose=False)
-        # Some architectures (arm) require machine type to be always set
-        failed_pattern = r'(?:kvm_init_vcpu.*failed)|(?:machine specified)'
+        # Some architectures (arm,power) require machine type to be always set
```

Comment:
PPC doesn't require machine type is always set, hardware/firmware restriction cause we need to use the workaround machine, so I think need to update the comment.

Reply:
Yes, I agree, corrected.